src/test/java/com/avispl/symphony/dal/communicator/lg/lcd/LgLCDControlTest.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (c) 2021 AVI-SPL, Inc. All Rights Reserved.
+ */
+package com.avispl.symphony.dal.communicator.lg.lcd;
+
+import static org.mockito.Mockito.times;
+
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+
+import com.avispl.symphony.api.dal.dto.control.ControllableProperty;
+import com.avispl.symphony.dal.communicator.lg.lcd.LgLCDConstants.controlProperties;
+
+/**
+ * Unit test for Control of LgLCDDevice
+ *
+ * @author Harry
+ * @version 1.0
+ * @since 1.0
```

Comment:
It's not 1.0. Please check the other places as well

Reply:
I fixed it